cmd/kubeadm/app/phases/markcontrolplane/markcontrolplane.go

```diff
@@ -18,19 +18,27 @@ package markcontrolplane
 
 import (
 	"fmt"
+	"strings"
 
 	"k8s.io/api/core/v1"
 	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/kubernetes/cmd/kubeadm/app/constants"
 	"k8s.io/kubernetes/cmd/kubeadm/app/util/apiclient"
 )
 
-// MarkControlPlane taints the control-plane and sets the control-plane label
-func MarkControlPlane(client clientset.Interface, controlPlaneName string, taints []v1.Taint) error {
+const deprecatedSuffix = "(deprecated)"
```

Comment:
need we add `will be removed in 1.22` comments here?

Reply:
it's actually for 1.24+.

i guess it's optional to remove this constant, but the TODO is linked bellow.

> https://github.com/kubernetes/kubeadm/issues/2200

Reply:
the TODO about upgrades is the other linked issue.